study.md

```diff
@@ -22,8 +22,8 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, define ORM and explain why using an ORM is valuable.
 
 ```md
-<!-- your response here -->
-```
+its a way to move  objects of an application to tables.
```

Comment:
Object relational-mapping is a method of representing arbitrary data as objects in a programming language.